Tasks/VitaliyKravchenok/Task4/Task4.py

```diff
@@ -0,0 +1,47 @@
+while True:
+
+    len_string = input("Input the length of the string, its value must not be less than 35: ")    
+
+    if not len_string.isdigit():
+        print("Incorrect input! Please, input digit value!")
+        continue
+
+    len_string = int(len_string)
+
+    if len_string < 35:
+        print("Incorrect input! Please, input digit value not less than 35!")
+        continue
+
+    break
+
+with open("text.txt", 'r', encoding= "utf8-r") as f:
+    rline = f.readlines()
+    with open("new_text.txt", 'w', encoding= "utf8-r") as f:             
```

Comment:
you open two files simultaneously, bur read the first file in memory as a whole, this does not make much sense, would be better to process the file at least line by line instead